pkg/autodiscovery/common/types/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/DataDog/datadog-agent/pkg/autodiscovery/common/types
+
+go 1.20
+
+replace github.com/DataDog/datadog-agent/pkg/util/log => ../../../util/log/
+
+require (
+	github.com/DataDog/datadog-agent/pkg/util/log v0.0.0-00010101000000-000000000000
+	github.com/stretchr/testify v1.8.4
+)
+
+require (
+	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.49.0-rc.2 // indirect
```

Comment:
Don't you need a replace for `scrubber` too ?

Reply:
Good catch!